### PR TITLE
Add section 'Editor and IDE Integrations'

### PR DESCRIPTION
**Pull request by KeyboardInterrupt**
This adds a Section on Editors and IDE Integrations for Ansible, it's based on personal experience, and inspired by this Page from the official Documentation: [other_tools_and_programs](https://github.com/ansible/ansible/blob/2797dc644aa8c809444ccac64cad63e0d9a3f9fe/docs/docsite/rst/community/other_tools_and_programs.rst)

The official Documentation does recommend some general Purpose Extensions and specific Editors, but those are either not Ansible specific enough, or outdated, those I have omitted.

I personally use VS Code, and the official Ansible Language Support Extension, if anyone else wants to add to the Section, for their favorite IDE or Editor, please feel free to do so!

As more Extensions for multiple different Editors might be added, we might want to split the Section up into Editor/IDE Specific sections, for now, I find it to be enough to start the Link Title with the Editor for which the Integration is, to both, make it easy to visually grep for a specific Editor, and to sort/group multiple Items for an Editor together.

### CODE REVIEW

**Comment by KeyboardInterrupt** [comment]
https://github.com/pearofducks/ansible-vim appears to be an maintained Ansible plugin for vim, does anyone know if this Extension is Awesome, and should be added to the Section?

**Comment by KeyboardInterrupt** [comment]
https://github.com/ansible-community/awesome-ansible/pull/56 <- this PR, should probably be added, to this Section as well, once it has been reviewed, or rather the Extension has been checked out.

**Comment by KeyboardInterrupt** [comment]
https://www.npmjs.com/package/@yaegassy/coc-ansible This is another Plugin for vim, utilizing the Ansible Language Server, which makes it a very promising plugin, does anyone know if it is Awesome, and thus should be added to the Section?

**Comment by KeyboardInterrupt** [comment]
https://github.com/MSDehghan/AnsiblePlugin this one, for idea/IntelliJ, has not been updated in a while, with the last update, 17 month ago, bringing FQCN support.
FQCN support is a huge plus, for being awesome ^^, as Ansible 2.9 will be a thing of the Past rather soon anyway.

File: README.md [hunk 1]
```diff
@@ -113,3 +114,11 @@
 - [DebOps](https://docs.debops.org/en/master/) - A extensive collection of Debian based Ansible Playbooks.
 - [ansible-ssm](https://github.com/HQarroum/ansible-ssm) - An ansible role to provision physical and virtual hosts with the AWS SSM agent.
 - [BlueBanquise](https://github.com/bluebanquise/bluebanquise) - An ansible coherent roles collection to deploy clusters.
+
+## Editor and IDE Integrations
+
+> Awesome Integrations into Text Editors and IDE's to make development with/for Ansible easier.
+
+- [Ansible Language Server](https://github.com/ansible/ansible-language-server) - Language Server that adds support for Ansible, to compatible Editors.
```
Comment (ganeshrn) [inline README.md:122]: Can add https://github.com/yaegassy/coc-ansible as vim lsp client for Ansible

Comment (ssbarnea) [reply README.md:122]: We also need to distinguish between the official/supported vscode extension and the other 3rd party implementations using ALS.

**Comment by KeyboardInterrupt** [comment]
resolved in #78

**Pull request closed without merging**